cloudsigma/resource_cloudsigma_server.go

```diff
@@ -395,9 +440,12 @@ func resourceCloudSigmaServerUpdate(ctx context.Context, d *schema.ResourceData,
 		updateRequest.Tags = expandTags(v.(*schema.Set).List())
 	}
 
-	err = stopServer(ctx, client, d.Id())
-	if err != nil {
-		return diag.FromErr(err)
+	if needRestart {
+		log.Printf("[DEBUG] Stopping the server")
```

Comment:
we log a server operation here https://github.com/cloudsigma/terraform-provider-cloudsigma/blob/master/cloudsigma/resource_cloudsigma_server.go#L528, so we can omit this log line